azext_iot/central/_help.py

```diff
@@ -0,0 +1,178 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+
+def load_central_help():
+    helps[
+        "iot central"
+    ] = """
+        type: group
+        short-summary: Manage Azure Central (IoTC) solutions & infrastructure
+    """
+
+    _load_central_devices_help()
+    _load_central_device_templates_help()
+
+
+def _load_central_devices_help():
+    helps[
+        "iot central app device"
+    ] = """
+        type: group
+        short-summary: Manage and configure IoTC devices
+    """
+
+    helps[
+        "iot central app device create"
+    ] = """
+        type: command
+        short-summary: Create a device in IoTC
+
+        examples:
+        - name: Create a device
+          text: >
+            az iot central app device create
+            --app-id {appid}
+            --device-id {deviceid}
```

Comment:
Good to keep an eye out for shared (cross service) parameters wherever we can and use them (rather than re-invent)...in this case `--device-id`.